token

```diff
@@ -5,6 +5,6 @@ DIR=`readlink -f $DIR`
 
 source $DIR/config
 
-ACCESS_TOKEN=`curl --insecure -s --data "client_id=admin-cli&username=admin&password=admin&grant_type=password" $KEYCLOAK_URL/realms/master/protocol/openid-connect/token | jq -r '.access_token'`
+ACCESS_TOKEN=`curl --key ./openshift-basedir/kube-apiserver/ca.key --cert ./openshift-basedir/kube-apiserver/ca.crt --cacert ./openshift-basedir/kube-apiserver/ca.crt -s --data "client_id=admin-cli&username=admin&password=admin&grant_type=password" $KEYCLOAK_URL/realms/master/protocol/openid-connect/token | jq -r '.access_token'`
```

Comment:
This is wrong, if you need a client cert, you make a client cert, using a CA key is absolutely not OK.

Reply:
This one has just been removed. It is no longer needed.